plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/navigation/service/JavaNavigationServiceImpl.java

```diff
@@ -237,23 +247,39 @@ public void getContent(
   }
 
   @Override
-  public Promise<ImplementationsDescriptorDTO> getImplementations(
-      Path project, String fqn, int offset) {
-    final String url =
-        appContext.getWsAgentServerApiEndpoint()
-            + "/java/navigation/implementations"
-            + "?projectpath="
-            + project.toString()
-            + "&fqn="
-            + fqn
-            + "&offset="
-            + offset;
-
-    return requestFactory
-        .createGetRequest(url)
-        .header(ACCEPT, APPLICATION_JSON)
-        .loader(loaderFactory.newLoader())
-        .send(unmarshallerFactory.newUnmarshaller(ImplementationsDescriptorDTO.class));
+  public Promise<ImplementationsDescriptor> findImplementations(
```

Comment:
So far, we've added jdt.ls extension services to JavaLanguageServiceExtensionClient. Unless there is a good reason, I would suggest to move this method ther.